compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__echo__echo_in_pipeline_printing_a_function_2.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: compiler-core/src/type_/tests/echo.rs
+expression: "\npub fn main() {\n  [1, 2, 3]\n  |> echo wibble\n}\n\nfn wibble(_: List(Float)) -> List(String) { todo }\n"
+---
+error: Type mismatch
+  ┌─ /src/one/two.gleam:4:6
+  │
+4 │   |> echo wibble
+  │      ^^^^^^^^^^^ This function does not accept the piped type
```

Comment:
I don't think we should be calling `echo` a function.

Reply:
Ooh this slipped me, it should only point to `wibble` in this case I think

Reply:
oooooooh so that's what the error is

Reply:
Ah wait you can't this is an old snapshot that I forgot to delete I think!

Reply:
Thank you.

It would be cool if insta could clean up old snapshots